AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Task/AWSAuthDeleteUserTask.swift

```diff
@@ -66,6 +67,19 @@ class AWSAuthDeleteUserTask: AuthDeleteUserTask, DefaultLogger {
         }
     }
 
+    private func signOutIfUserWasNotFound(with error: Error) async {
+        guard case AuthError.service(_, _, let underlyingError) = error,
+              case .userNotFound = (underlyingError as? AWSCognitoAuthError) else {
+            return
+        }
+        log.verbose("User not found, signing out")
+        let signOutData = SignOutEventData(
+            globalSignOut: true)
```

Comment:
```suggestion
        let signOutData = SignOutEventData(globalSignOut: true)
```
nit: looks like a small line